requirements.txt

```diff
@@ -9,7 +9,7 @@ gunicorn==19.8.1
 idna==2.6
 mccabe==0.6.1
 oauthlib==2.1.0
-psycopg2==2.7.4
+psycopg2-binary==2.7.7
```

Comment:
IMO this can be made a separate issue for beginners since your commit is in no way related to this change.

Reply:
Remove this line from your commit.